config/routes.rb

```diff
@@ -3,6 +3,7 @@
   namespace :api do
     namespace :v1 do
       get '/books/search', to: 'books#search'
+      get '/quote', to: 'quotes#index'
```

Comment:
I was thinking that because we are only returning a single quote it might be more RESTful to direct this through the controller show action. But then I realized that would be if we were searching it by ID. So I think this is RESTful and good.